### DISCUSSION TITLE
[Common Question] Where are the chat sessions stored? Can I view them in Obsidian? Can I remove the chat sessions (one by one or in bulk)?

### DISCUSSION THREAD

**Discussion by romeoleung** [brianpetro/obsidian-smart-connections#122] [Q&A] [answered]
I was testing out the functionalities of Smart Connections, and I created some chats.
After several different conversations, I click on the Chat History button, and can see the list of chats I did listed out.
I wanted to see where they are in my vault, want to remove some of them or all of them, but I can't find a way to do so. 
So, the questions I have are:

1. Where are the chat sessions stored?
2. Can I view the chat sessions in Obsidian (as files, not from the chat window), or I'll have to open them externally?
3. How to bulk delete the chat sessions?

Thank you!

**Comment by brianpetro** [comment, marked as answer]
Hi @romeoleung and welcome to the Smart Connections community.

Look for the history in the `.smart-connections/chats` folder. It's hidden, so you'll have to use your normal file explorer to find it. Deleting from the UI will come soon.

I hope that clears things up for you.

Brian 🌴

**Reply by romeoleung** [reply]
Saw the files in `.smart-connections/chats` folder. This answers my questions.
Looking forward for the chat history management from UI to come out!
Thank you for the warm messages!
